tests/unit_tests/main_test.js

```diff
@@ -40,6 +40,114 @@ context("HintCoordinator", () => {
   });
 });
 
+context("Next zoom level", () => {
+  // NOTE: All these tests use the Chrome zoom levels, which are the default!
+  should("Zoom in 0 times", async () => {
+    const count = 0;
+    const currentZoom = 1.00;
+    const nextZoom = await nextZoomLevel(currentZoom, count);
+    assert.equal(1.00, nextZoom);
+  });
+
+  should("Zoom in 1", async () => {
+    const count = 1;
+    const currentZoom = 1.00;
+    const nextZoom = await nextZoomLevel(currentZoom, count);
+    assert.equal(1.10, nextZoom);
+  });
+
+  should("Zoom out 1", async () => {
+    const count = -1;
+    const currentZoom = 1.00;
+    const nextZoom = await nextZoomLevel(currentZoom, count);
+    assert.equal(0.90, nextZoom);
+  });
+
+  should("Zoom in 2", async () => {
+    const count = 2;
+    const currentZoom = 1.00;
+    const nextZoom = await nextZoomLevel(currentZoom, count);
+    assert.equal(1.25, nextZoom);
+  });
+
+  should("Zoom out 2", async () => {
+    const count = -2;
+    const currentZoom = 1.00;
+    const nextZoom = await nextZoomLevel(currentZoom, count);
+    assert.equal(0.80, nextZoom);
+  });
+
+  should("Zoom in from between values", async () => {
+    const count = 1;
+    const currentZoom = 1.05;
+    const nextZoom = await nextZoomLevel(currentZoom, count);
+    assert.equal(1.10, nextZoom);
+  });
+
+  should("Zoom out from between values", async () => {
+    const count = -1;
+    const currentZoom = 1.05;
+    const nextZoom = await nextZoomLevel(currentZoom, count);
+    assert.equal(1.00, nextZoom);
+  });
+
+  should("Zoom in past the maximum", async () => {
+    const count = 15;
+    const currentZoom = 1.00;
+    const nextZoom = await nextZoomLevel(currentZoom, count);
+    assert.equal(5.00, nextZoom);
+  });
+
+  should("Zoom out past the minimum", async () => {
+    const count = -15;
+    const currentZoom = 1.00;
+    const nextZoom = await nextZoomLevel(currentZoom, count);
+    assert.equal(0.25, nextZoom);
+  });
+
+  should("Zoom in from below the minimum", async () => {
```

Comment:
These next four tests ensure zooming works when we zoom from outside of Chromes min/max zoom range, because Chrome (not Firefox) allows the zoom number to be outside the default accepted range. Supporting this should help avoid issues with other Chromium based browsers that allow different zoom values.